composer.json

```diff
@@ -41,6 +29,8 @@
         "drupal/media_entity": "^1.6",
         "drupal/media_entity_image": "^1.2",
         "drupal/bootstrap": "^3.1",
+        "islandora/islandora": "dev-8.x-1.x",
+        "islandora/claw-jsonld": "dev-8.x-1.x",
```

Comment:
Duh, shoulda known that. I just pushed another commit that removes islandora and claw-jsonld. I did a fresh `composer install` and `web/modules/contrib` contained islandora, islandora_collection and claw-jsonld so should be good to go.